lms/static/js/student_account/views/LoginView.js

```diff
@@ -132,8 +132,7 @@
                         msg = gettext('An error has occurred. Try refreshing the page, or check your Internet connection.');
                     }
                     this.errors = ['<li>' + msg + '</li>'];
-                    this.setErrors();
-                    this.element.hide(this.$resetSuccess);
+                    this.$formFeedback.find('.submission-success').remove();
```

Comment:
I plan to clean up these lines, just haven't decided on the best way to do so just yet.